src/averageAges.js

```diff
@@ -55,7 +80,40 @@ function calculateWomenAverageAge(people, withChildren) {
  * @return {number}
  */
 function calculateAverageAgeDiff(people, onlyWithSon) {
-  // write code here
+  const result = [];
+
+  const getMotherAge = (motherName) => {
+    let wasBorn;
+
+    people.find(mother => {
+      if (mother.name === motherName) {
+        wasBorn = mother.born;
+      }
+    });
+
+    return wasBorn;
+  };
```

Comment:
you can do it a lot easier, without redundant `let`, you're using `find` wrong
```suggestion
  const getMotherAge = (motherName) => {
    const foundMother = people.find(mother => {
      return mother.name === motherName
    });
    
    return foundMother ? foundMother.born : null;
  };
```